src/app/Plans/components/PlansTable.tsx

```diff
@@ -49,6 +61,16 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
 
   const getMigration = (plan) => migrations.filter((migration) => migration.plan === plan);
 
+  const ratioVMs = (plan) => {
```

Comment:
Sorry I didn't catch this before. Can we add a type to the arguments of these functions?
`const getMigration = (plan: IPlan) => { ...`
`const ratioVMs = (plan: IPlan) => { ...`

I wonder if we should add `noImplicitAny` in our tsconfig... maybe later.

Reply:
@mturley, done. 

+1 for `noImplicityAny`